### ISSUE TITLE
Log names of other interactions

### ISSUE THREAD

**Issue by 2br-2b** [Apollo-Roboto/discord.py-ext-prometheus#21]
Right now, the interaction name is only logged if it's a command. This gives no visibility into what buttons are being pressed, what modals are being submitted, etc. It would be helpful to provide more visibility here

**Comment by ApolloRoboto** [comment]
I totally agree. At the moment, all interractions are under one metric but that might not be the best approach. We might need new counters to support modal names and button names properly

**Comment by ApolloRoboto** [comment]
`discord_event_on_application_command_interaction` could be splitted into the 5 [interaction types](https://discordpy.readthedocs.io/en/stable/interactions/api.html?highlight=interactiontype#discord.InteractionType)

- `discord_interaction_component`
- `discord_interaction_slash_command`
- `discord_interaction_modal_submit`
- `discord_interaction_ping`
- `discord_interaction_autocomplete`
